.gitignore

```diff
@@ -54,6 +54,15 @@ tags
 /doc/tutorial/workbooks/*.dvi
 /doc/tutorial/workbooks/*.bbl
 /doc/tutorial/workbooks/*.blg
+/doc/sphinx/build
+/doc/sphinx/source/*.rst
+/doc/sphinx/source/platforms/*.rst
+/doc/sphinx/source/technotes/*.rst
+/doc/sphinx/source/modules
+!/doc/sphinx/source/index.rst
+!/doc/sphinx/source/technotes/index.rst
+!/doc/sphinx/source/platforms/index.rst
```

Comment:
Exclude from .gitignore.

So `/doc/sphinx/source/*.rst` + `!/doc/sphinx/source/index.rst` ignores all rst files in `source`, except for `index.rst`.